client/my-sites/plugins/main.jsx

```diff
@@ -55,7 +55,7 @@ export default React.createClass( {
 	mixins: [ URLSearch, PluginNotices ],
 
 	getInitialState() {
-		return this.getPluginsState( this.props );
+		return Object.assign( this.getPluginsState( this.props ), { bulkManagement: [] } );
```

Comment:
This is related to something that have been in my head for some time... Why are we storing the selection state of each plugin in our data stores? Wouldn't it make much more sense to just have it in the state of the main component? after all, all the bulk actions are being initiated by components that descend from it, and every time we need to do any action in all the selected plugins, we are retrieving that data from the store and sending individual actions for each one ... so... why not placing it directly in (for example) the state of `plugins/main.jsx`?